simcon_project/conversation_templates/views/conversation.py

```diff
@@ -131,7 +131,7 @@ def conversation_step(request, ct_node_id):
 
     # GET request
     choice_form = TemplateNodeChoiceForm(ct_node=ct_node)
-    ct = ct_node.parent_template
+    ct = ct_node.parent_template_node
```

Comment:
This should still be parent_template since its a template node object and not a template node choice object

Reply:
Great catch!